tests/test_filters.py

```diff
@@ -404,15 +440,22 @@ def _assert_column_definition(
         column_name,
         column_type,
         column_type_length,
-        column_attributes
+        column_attributes,
+        column_type_values=None
```

Comment:
In your changes, the time to use `_assert_column_definition_with_length` is when the type is `enum` or `set`. `_assert_column_definition_with_value` is used in the rest of the cases. The caller of these function should already know which type it'd like to verify and whether the type has `ColumnTypeLength` or `ColumnTypeValue`.